docs/api/objects/post.md

```diff
@@ -2,21 +2,25 @@
 
 ### Structure
 
-<!-- deno-fmt-ignore-start -->
-| Field | Type | Description | Optional |
-| - | - | - | - |
-| _id | UUID4 | The post's UUID4. | |
-| post_id | UUID4 | The post's UUID4. | |
-| post_origin | string/UUID4 | Where the post was created. Can be `"home"`, `"inbox"`, `"livechat"`, or a chat's UUID4. | |
-| type | integer | The post's type. | |
-| t | [Extended Timestamp](./extended-timestamp) object | The post's creation timestamp. | |
-| u | string | The username of the post author. | |
-| p | string | The content of the post. | |
-| unfiltered_p | string | The non-filtered content of the post. | ✓ |
-| pinned | boolean | Whether the post is pinned. | |
-| isDeleted | boolean | Whether the post is deleted. | |
-| mod_deleted | boolean | Whether the post was deleted by a moderator. | ✓ |
-| deleted_at | integer | The post's deletion timestamp in Unix seconds. | ✓ |
+<!-- deno-fmt-ignore-start --> <!-- Uploads was down -->
+| Field        | Type                                              | Description                                                                              | Optional |
```

Comment:
Could this be formatted the way all the other tables are?

Reply:
no, it's unreadable and uneditable otherwise

Reply:
How? Also, that's what the rest of the tables do, it should really be consistent

Reply:
(tried to do it with 1 other but the ext failed to format that one)

Reply:
> I had to format it with a vscode ext so I could actually edit it

What do you mean by that? You can edit it when it's formatted without fifty million spaces and dashes

Reply:
Its uneditable to me because the formatting is inreadable when compressed

Reply:
Compressed?

Reply:
like unformatted

Reply:
I really don't see what you mean

_"Formatted"_
![image](https://github.com/user-attachments/assets/d4529da4-d31f-4682-baab-bc9a24f75eb2)

_Current_
![image](https://github.com/user-attachments/assets/180a8b8b-10af-4990-945d-b17d9379eeb8)

Reply:
gh wraps lines

Reply:
Yes?

Reply:
Meaning it looks worse with line wrapping because it adds an extra line inbetween lines

Reply:
Even without word wrapping (but especially with word wrapping which is pretty much necessary for md files), the current is way easier to read and has more information on screen

_"Formatted"_
![image](https://github.com/user-attachments/assets/53790e86-175e-44a5-821c-301c5227c70a)

_"Current"_
![image](https://github.com/user-attachments/assets/ba3d0698-6142-497a-9b7f-92763409f7ba)